### ISSUE TITLE
Register user with additionnal fields

### ISSUE THREAD

**Issue by 2kne** [knpuniversity/oauth2-client-bundle#262]
Hello, I would like to register with google in two steps. 
Click btn -> Google connect 
a -> if user in db -> login user
b -> if !user in db -> 
    1. display -> form with additionnal fields (int, string etc) 
    2. Register user if form is valid

**Important: I want save user in db only after the second step.**

Is it possible ?
Thanks.

**Comment by DonCallisto** [comment]
This seems just a matter of your business logic. Nothing that cannot be already done. 
You should  take care of this by yourself and this is not an issue of this bundle - or the underlying library - from my POV.

**Comment by 2kne** [comment]
> This seems just a matter of your business logic. Nothing that cannot be already done.
> You should take care of this by yourself and this is not an issue of this bundle - or the underlying library - from my POV.
